SQL/9999-99-99-drop_tables.sql

```diff
@@ -0,0 +1,163 @@
+DROP TABLE IF EXISTS `physiological_electrode_type`;
+DROP TABLE IF EXISTS `physiological_channel`;
+DROP TABLE IF EXISTS `physiological_channel_type`;
+DROP TABLE IF EXISTS `physiological_status_type`;
+DROP TABLE IF EXISTS `physiological_parameter_file`;
+DROP TABLE IF EXISTS `physiological_file`;
+DROP TABLE IF EXISTS `physiological_output_type`;
+DROP TABLE IF EXISTS `physiological_modality`;
+
+-- 0000-00-04-Help.sql
+DROP TABLE IF EXISTS `help`;
+
+-- 0000-00-03-ConfigTables.sql
+DROP TABLE IF EXISTS `Config`;
+DROP TABLE IF EXISTS `ConfigSettings`;
+
+-- 0000-00-02-Menus.sql
+DROP TABLE IF EXISTS `LorisMenuPermissions`;
+DROP TABLE IF EXISTS `LorisMenu`;
+
+-- 0000-00-01-Permission.sql
+DROP TABLE IF EXISTS `user_perm_rel`;
+DROP TABLE IF EXISTS `notification_modules_perm_rel`;
+DROP TABLE IF EXISTS `permissions`;
+DROP TABLE IF EXISTS `permissions_category`;
+
+
+-- 0000-00-00-schema.sql
+DROP TABLE IF EXISTS `candidate_consent_rel`;
+DROP TABLE IF EXISTS `consent`;
+DROP TABLE IF EXISTS `candidate_consent_history`;
+
+DROP TABLE IF EXISTS `acknowledgements`;
+
+DROP TABLE IF EXISTS `data_release_permissions`;
+DROP TABLE IF EXISTS `data_release`;
+
+DROP TABLE IF EXISTS `ExternalLinks`;
+DROP TABLE IF EXISTS `ExternalLinkTypes`;
+
+DROP TABLE IF EXISTS `feedback_mri_comments`;
+DROP TABLE IF EXISTS `feedback_mri_predefined_comments`;
+DROP TABLE IF EXISTS `feedback_mri_comment_types`;
+DROP TABLE IF EXISTS `feedback_bvl_entry`;
+DROP TABLE IF EXISTS `feedback_bvl_thread`;
+DROP TABLE IF EXISTS `feedback_bvl_type`;
+
+DROP TABLE IF EXISTS `genomic_cpg`;
+DROP TABLE IF EXISTS `genomic_cpg_annotation`;
+DROP TABLE IF EXISTS `genomic_sample_candidate_rel`;
+DROP TABLE IF EXISTS `genomic_candidate_files_rel`;
+DROP TABLE IF EXISTS `genomic_files`;
+DROP TABLE IF EXISTS `genomic_analysis_modality_enum`;
+DROP TABLE IF EXISTS `GWAS`;
+DROP TABLE IF EXISTS `CNV`;
+DROP TABLE IF EXISTS `SNP_candidate_rel`;
+DROP TABLE IF EXISTS `SNP`;
+DROP TABLE IF EXISTS `genotyping_platform`;
+DROP TABLE IF EXISTS `gene`;
+DROP TABLE IF EXISTS `genome_loc`;
+
+DROP TABLE IF EXISTS `parameter_session`;
+DROP TABLE IF EXISTS `parameter_file`;
+DROP TABLE IF EXISTS `parameter_candidate`;
+DROP TABLE IF EXISTS `parameter_type_override`;
+DROP TABLE IF EXISTS `parameter_type_category_rel`;
+DROP TABLE IF EXISTS `parameter_type_category`;
+DROP TABLE IF EXISTS `parameter_type`;
+
+DROP TABLE IF EXISTS `issues_watching`;
+DROP TABLE IF EXISTS `issues_comments_history`;
+DROP TABLE IF EXISTS `issues_history`;
+DROP TABLE IF EXISTS `issues_comments`;
+DROP TABLE IF EXISTS `issues`;
+DROP TABLE IF EXISTS `issues_categories`;
+
+DROP TABLE IF EXISTS `media`;
+DROP TABLE IF EXISTS `server_processes`;
+DROP TABLE IF EXISTS `StatisticsTabs`;
+DROP TABLE IF EXISTS `user_login_history`;
+DROP TABLE IF EXISTS `user_account_history`;
+DROP TABLE IF EXISTS `data_integrity_flag`;
+DROP TABLE IF EXISTS `certification_training_quiz_answers`;
```

Comment:
The training ones are going to conflict with #4535

Reply:
@driusan merge this one first :)